src/plugins/intel_gpu/src/graph/paged_attention.cpp

```diff
@@ -21,17 +21,26 @@ layout paged_attention_inst::calc_output_layout(const paged_attention_node& /*no
 
 template<typename ShapeType>
 std::vector<layout> paged_attention_inst::calc_output_layouts(paged_attention_node const& /*node*/, kernel_impl_params const& impl_param) {
-    auto data_layout = impl_param.get_input_layout(0);
+    const auto& q_layout = impl_param.get_input_layout(0);
+    const auto& desc = impl_param.typed_desc<paged_attention>();
+    auto data_layout = q_layout;
+
+    if (desc->k_head_size != desc->v_head_size) {
+        auto data_shape = { q_layout.get_partial_shape()[0],
+                            ov::Dimension(desc->heads_num),
+                            ov::Dimension(desc->v_head_size) };
```

Comment:
PagedAttention has 2D output [batch_size_in_tokens, num_heads * head_size], so it should be like `[q_layout.get_partial_shape()[0], desc->heads_num * desc->v_head_size]`